formal/data_ind_timing/run.sby.j2

```diff
@@ -0,0 +1,23 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+[tasks]
+slow sl
+fast fs
+single si
+
+[options]
+mode bmc
+depth 50
```

Comment:
Wow, that's a long way! Do we actually need a depth of 50? I assume that a single operation only takes a few cycles, no?

Reply:
Only multiplication operations are fast. A division needs 37 cycles plus some init cycles, that would be then around 40, so to round it up I landed on 50.
It might be possible to separate this, but I wanted to keep it simple. And it does not compute for so long.

Reply:
Ahh! That makes sense. :+1: